client/views/msg_preview_filesize.tpl

```diff
@@ -0,0 +1,5 @@
+{{#preview}}
+    {{#if showSize}}
+        <span class="preview-size">({{friendlysize size}})</span>
```

Comment:
Add a space before opening parenthesis? Looks kinda weird when there's no whitespace between the link and the file size.